controllers/dspipeline_controller.go

```diff
@@ -259,7 +253,7 @@ func (r *DSPAReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 // Clean Up any resources not handled by garbage collection, like Cluster ResourceRequirements
-func (r *DSPAReconciler) cleanUpResources(ctx context.Context, req ctrl.Request, dsp *dspav1alpha1.DataSciencePipelinesApplication, params *DSPAParams) error {
+func (r *DSPAReconciler) cleanUpResources(params *DSPAParams) error {
 	err := r.CleanUpCommon(params)
```

Comment:
since we're simplifying here, could we also just return the result here instead of checking if `err` is nil, just to return `nil` if it is?